apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1511,7 +1511,7 @@ export class RushConfiguration {
       this._commonVersionsConfigurations = new Map();
     }
 
-    const variantKey: string = variant || '';
+    const variantKey: string = variant || 'default';
```

Comment:
Should this go in `RushConstants`?

Reply:
Changed my mind on this. Going to use an empty string and update the comment. It's possible for us to conflict with a variant named `default` but not with an empty string, since you can't create a variant with that name.